advanced/bash-completion/pihole

```diff
@@ -7,7 +7,7 @@ _pihole() {
 
     case "${prev}" in
         "pihole")
-            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex restartdns status tail uninstall updateGravity updatePihole version wildcard arpflush api"
+            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex reloaddns reloaddns-lists status tail uninstall updateGravity updatePihole version wildcard arpflush api"
```

Comment:
If the command `reloaddns-lists` only reload lists and keep the cache, I suggest a different name - `reloadlists`:

```suggestion
            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex reloaddns reloadlists status tail uninstall updateGravity updatePihole version wildcard arpflush api"
```